app/views/static_pages/index.html.erb

```diff
@@ -17,20 +17,30 @@
 </div>
 
 <div id="tab-cuisines" style="display:none;">
-  <div class="col-xs-3" class="filter-box">
-    <div>Chinese</div>
-    <div>Japanese</div>
-    <div>Westen</div>
+  <div class="col-xs-12">
```

Comment:
If you want to nest columns inside columns with the Bootstrap grid, you would better use a .row in between:

```
<div class="col-xs-6">
    <div class="row">
        <div class="col-xs-4"></div>
        <div class="col-xs-4"></div>
        <div class="col-xs-4"></div>
    </div>
</div>
```

In your case, I suggest you just replace the 2 ```.col-xs-12``` by ```.row```